examples/image-to-text/requirements_vlm_eval.txt

```diff
@@ -0,0 +1,6 @@
+lm-eval==0.4.7
+datasets==2.21.0
+evaluate == 0.4.3
+rouge_score == 0.1.2
+accelerate < 0.34.0
```

Comment:
Is this a hard constraint? This won't work as we now need https://github.com/huggingface/optimum-habana/blob/5448802fb0590f471db3a79356120633447ed6fa/setup.py#L35